### PR TITLE
Create t3iii-eye-protection.md

### PR DESCRIPTION
**Pull request by The-Kergan**
@geoffwasham 
@Nelviticus you can approve, but would need access to the dropbox folder to try and upload images/pdfs - I have a help req. open with GitHub since I am unable to upload files - I assume it must be my router, because I've tried everything I can think of on two different computers.

Files that need to be uploaded to Github 'Public' folder:
```
Dropbox\COVID-19\1-WORK OUTPUT\WEEK 3\FINAL\T3iii-Eye_Protection_v3.pdf
Dropbox\COVID-19\5-Images\Infographics\PPE-Sequence.jpg
Dropbox\COVID-19\5-Images\Infographics\PPE-Sequence_1.jpg
```

### CODE REVIEW

File: docs/tools/t3iii-eye-protection.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+- Remove a face shield or goggles by removing it from behind, pulling the straps over the head and allowing the goggles or shield to fall forward, away from you
+  - **Do not** touch the front of the goggles or face shield as it may be contaminated with COVID-19
+
+
+![Sequence of taking off PPE](PPE-Sequence_1.jpg)
+**Source:** [CDC](https://www.cdc.gov/hai/pdfs/ppe/PPE-Sequence.pdf)
+
+#### Reusing Eye Protection
+- Eye protection must be cleaned between uses:
+  1. Hand wash with hot water (greater than 70°C/160°F) and dish soap for about 1 minute, then rinse and dry before disinfecting with store bough disinfectant cleaning solution or disinfecting wipes
```
Comment (Nelviticus) [inline docs/tools/t3iii-eye-protection.md:78]: 'store bough**t** disinfectant' (missing 't')

Comment (The-Kergan) [inline docs/tools/t3iii-eye-protection.md:78]:
```suggestion
  1. Hand wash with hot water (greater than 70°C/160°F) and dish soap for about 1 minute, then rinse and dry before disinfecting with store bought disinfectant cleaning solution or disinfecting wipes
```

**Comment by The-Kergan** [comment]
@Nelviticus I fixed 'bought' typo.

File: docs/tools/t3iii-eye-protection.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+**Somewhat Effective:** Wrap-around sunglasses, safety glasses
+**No evidence that they are effective:** Prescription glasses, normal sunglasses
+**Not Effective:** No eye protection
+
+#### Should I Wear Eye Protection?
+**When leaving the house – Yes.**
+**When at home and you are symptomatic or have had recent contact with someone with COVID-19 – Yes.**
+**When at home and you have been in lock-down for greater than 14 days and no one at home has symptoms – No.**
+
+There is no harm in wearing eye protection, and it is believed to prevent droplets and a gaseous cloud containing the virus traveling up to 8 meters / 27 feet from reaching the eyes, which can cause infection.
```
Comment (Nelviticus) [inline docs/tools/t3iii-eye-protection.md:20]: Remove comma before 'and'

File: docs/tools/t3iii-eye-protection.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+- COVID-19 is transmitted in droplets or a gaseous cloud exhaled by an individual infected with COVID-19
+  - These droplets can travel 2 meters or 6 feet away from the individual
+  - A gaseous cloud can travel for up to 8 meters / 27 feet
+  
+**Source:** [JAMA](https://jamanetwork.com/journals/jama/fullarticle/2763852)
+
+- Eye protection will prevent or reduce COVID-19 transmission through droplets contacting the eyes
+  - Eye protection also discourages eye-rubbing, which also protects you against COVID-19
+- Eye protection is **only effective** when used in combination with other personal protective equipment (PPE), frequent hand washing/sanitizing, social distancing, and avoiding face-touching
```
Comment (Nelviticus) [inline docs/tools/t3iii-eye-protection.md:30]: Remove comma before 'and'

File: docs/tools/t3iii-eye-protection.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  <iframe width="1280" height="720" src="https://www.youtube.com/embed/RTbwcZjrPag" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+<div class="video-responsive">
+  <iframe width="1280" height="720" src="https://www.youtube.com/embed/owiAiM03DJE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+  - Plastic safety goggles may also be worn by hospital staff when face shields are limiting or when using microscopes [Public Health Ontario](https://www.publichealthontario.ca/-/media/documents/ncov/updated-ipac-measures-covid-19.pdf?la=en)
+- Safety goggles or wrap-around sunglasses can provide limited eye protection
+- Prescription glasses or normal sunglasses have large gaps between the lenses and the face, and thus provide inadequate protection
```
Comment (Nelviticus) [inline docs/tools/t3iii-eye-protection.md:53]: Remove comma before 'and'

File: docs/tools/t3iii-eye-protection.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  - There is no evidence that normal glasses provide significant protection against COVID-19
+
+**Source:** [CDC – see Eye Protection](https://www.cdc.gov/coronavirus/2019-ncov/hcp/infection-control-recommendations.html?CDC_AA_refVal=https%3A%2F%2Fwww.cdc.gov%2Fcoronavirus%2F2019-ncov%2Finfection-control%2Fcontrol-recommendations.html), [UK Government](https://www.gov.uk/government/publications/wuhan-novel-coronavirus-infection-prevention-and-control/covid-19-personal-protective-equipment-ppe#ppe-guidance-by-healthcare-context), [Southern College of Optometry](https://tec.sco.edu/news/posts/contact-lens-wear-and-coronavirus-covid-19)
+
+#### Putting-On Eye Protection
+
+- Wash your hands with alcohol-based hand rub or soap and water for 20 seconds before putting on eye protection
+- Place your eye protection on your face **after** putting on your face mask and **before** putting on your gloves, and adjust to fit
```
Comment (Nelviticus) [inline docs/tools/t3iii-eye-protection.md:61]: Remove comma before 'and'

File: docs/tools/t3iii-eye-protection.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+
+![Sequence of taking off PPE](PPE-Sequence_1.jpg)
+**Source:** [CDC](https://www.cdc.gov/hai/pdfs/ppe/PPE-Sequence.pdf)
+
+#### Reusing Eye Protection
+- Eye protection must be cleaned between uses:
+  1. Hand wash with hot water (greater than 70°C/160°F) and dish soap for about 1 minute, then rinse and dry before disinfecting with store bought disinfectant cleaning solution or disinfecting wipes
+  2. Soak in 1-part bleach to 50 parts of hot water (greater than 70°C/160°F) for greater than 5 minutes, then rinse with water
+      - This means 2 tablespoons or 20mL of bleach per liter of water, or 1/3 of a cup bleach per gallon of water
```
Comment (Nelviticus) [inline docs/tools/t3iii-eye-protection.md:80]: Lower-case 'l' in 'ml'

**Pull request merged**